pkg/ipfs/screener.go

```diff
@@ -0,0 +1,241 @@
+// VulcanizeDB
+// Copyright © 2019 Vulcanize
+
+// This program is free software: you can redistribute it and/or modify
+// it under the terms of the GNU Affero General Public License as published by
+// the Free Software Foundation, either version 3 of the License, or
+// (at your option) any later version.
+
+// This program is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+// GNU Affero General Public License for more details.
+
+// You should have received a copy of the GNU Affero General Public License
+// along with this program.  If not, see <http://www.gnu.org/licenses/>.
+
+package ipfs
+
+import (
+	"bytes"
+
+	"github.com/ethereum/go-ethereum/common"
+	"github.com/ethereum/go-ethereum/core/types"
+	"github.com/ethereum/go-ethereum/rlp"
+
+	"github.com/vulcanize/vulcanizedb/pkg/config"
+)
+
+// ResponseScreener is the inteface used to screen eth data and package appropriate data into a response payload
+type ResponseScreener interface {
```

Comment:
Yeah I agree with a switch to `Filterer`! I went with "Response" because the seed node is filtering the data it syncs from geth to generate "responses" for each of its subscriptions, but that is pretty flimsy naming. Not sure what would be more appropriate, maybe "PayloadFilterer"?